ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/resources/streaming/QueryStreamWriterTest.java

```diff
@@ -85,9 +89,16 @@
   private QueryStreamWriter writer;
   private ByteArrayOutputStream out;
   private OutputNode.LimitHandler limitHandler;
+  private ObjectMapper objectMapper;
 
   @Before
   public void setUp() {
+
+    objectMapper = new ObjectMapper();
```

Comment:
Another part that duplicates the initialization code. Also - objectMappers don't need creating each test.